### PR TITLE
chore: deployment values for new playground

### LINKED ISSUE #441: chore: playground -  3rd party components configuration
we have to prepare kafka, rabbit, postgres and whatnot, config-wise

- [x] postgres
- [x] kafka (optional)
- [ ] rabbitmq (optional)
- [x] victoria metrix
- [x] run full test

configs:
generic.0 - postgres
generic.1 - kafka
generic.2 - victoria
+
kube.master.0
kube.node.0
kube.node.1

### PR DESCRIPTION
**Pull request by Kononnable**
Closes #441

### CODE REVIEW

File: deployment/helm/cdl/values.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-  imageRepositoryPath: epiphanyplatform/
-  imageVersion: master
+  imageRepositoryPath: epiphanyservices.azurecr.io/
+  imageVersion: develop
   imagePullPolicy: Always
   communicationMethod: "kafka"
-  kafkaBrokers: 10.1.5.4:9092
-  amqpConnectionString: "amqp://rabbitmq:rabbitmq@10.1.1.4:5672/%2f"
+  kafkaBrokers: 10.1.1.9:9092
+#  amqpConnectionString: "amqp://rabbitmq:rabbitmq@10.1.1.4:5672/%2f"
```
Comment (frondeus) [inline deployment/helm/cdl/values.yaml:7]: Is this comment left by accident?

**Comment by esavier** [comment]
yo, question, isn't this going to cdl-deploy?

File: deployment/helm/cdl/values.yaml (hunk 1 above)
Comment (Kononnable) [reply deployment/helm/cdl/values.yaml:7]: No, we don't have rabbitmq on the current playground, so to make it obvious that changing `communicationMethod` won't work without fixing the connection string.

**Comment by Kononnable** [comment]
Maybe it will go there, don't know.
Either way, moving deployment config to another repo is not part of mentioned issue.

**Review by smores56 — APPROVED** [review]

**Pull request merged**